src/backend/TrafficCourtsApi/ticket-worker/Features/Emails/EmailSender.cs

```diff
@@ -84,17 +92,20 @@ public async Task SendUsingTemplate(string to, string subject, TicketDisputeCont
 
                 emailModel.LogoImage = dataURIScheme("png", "ticket-worker.Features.Emails.Resources.bc-gov-logo.png");
 
-                var result = await _email
+                var email = _email
                     .To(to)
                     .Subject(subject)
-                    .UsingTemplateFromEmbedded("ticket-worker.Features.Emails.Resources.submissiontemplate.liquid", emailModel, this.GetType().GetTypeInfo().Assembly, true)
-                    .SendAsync();
+                    .UsingTemplateFromEmbedded("ticket-worker.Features.Emails.Resources.submissiontemplate.liquid", emailModel, this.GetType().GetTypeInfo().Assembly, true);
 
-                
-                if (!result.Successful)
+                if ( _emailFilter == null || _emailFilter.IsAllowed(to))
                 {
-                    _logger.LogError("Failed to send an email");
+                    var result = await email.SendAsync();
+                    if (!result.Successful)
+                    {
+                        _logger.LogError("Failed to send an email");
+                    }
                 }
+               
```

Comment:
I would also like log messages to indicate that the target email address was not allowed to be sent.  In addition, we should have a debug log message to indicate that the IsAllowed block returned true.